blocks/highlight-card/highlight-card.css

```
@@ -0,0 +1,34 @@
+.highlight-card-image {
```

All block styles should be scoped to the root block class to ensure there not class collisions.

eg.
```suggestion
.highlight-card.block .highlight-card-image {
```